### ISSUE TITLE
Ahmyth working

### ISSUE THREAD

**Issue by w1l5t3r** [AhMyth/AhMyth-Android-RAT#249]
Ahmyth wouldn't work and kept saying decompiling failed then building failed. Solved by uninstalling current java and installing version jre 1.80_181. This allows it to bind with an apk file and make sure the apk is under 3mb in size. 

link to apps that bind with ahmyth. 

[https://www.techuntold.com/smallest-apps/](url)

look at the app list and google search app names and download.

ahmyth does work!!  DO NOT UPDATE JAVA AFTER INSTALLING jre 1.80_181 or it will not work again! 

Link to screenshot of binding app 

[https://pasteboard.co/JxAhAeY.png](url)

**Comment by Morsmalleo** [comment]
yeah i have done this and i still get problems when binding with an apk, it does do anything upon selecting the original apk file to bind with, the black terminal on the bottom of the AhMyth GUI stays blank when it shouldnt
